### ISSUE TITLE
multi-head attention

### ISSUE THREAD

**Issue by tanjiarui** [AlexanderBogatko/Keras-DeTR#4]
there is a quick question. is the definition of multi-head same with the [layer](https://www.tensorflow.org/api_docs/python/tf/keras/layers/MultiHeadAttention) implemented by tf? if so, may i directly use it? thanks

**Comment by AlexanderBogatko** [comment]
Didn't quite understand the question, i already use standard TF multi-head attention (as part of [TransformerDecoderBlock](https://github.com/AlexanderBogatko/Keras-DeTR/blob/main/detr/blocks.py#L89), for example) - definition of layer didn't change since it appears in TF 2.4.0.

**Comment by tanjiarui** [comment]
yes, i understand, but the issue is that I'm training a model with over 600 loss at first. does it normal?
the data set distribution is much simpler than coco data set

**Comment by AlexanderBogatko** [comment]
If loss converges during training and model gives good result on your valid/test set - it is normal ) If not - something goes wrong. I haven't tested current code version, so i don't recommend fully rely on my implementation.
